src/Spd.Manager.Licence/ControllingMemberCrcAppContract.cs

```diff
@@ -42,30 +43,49 @@ public record ControllingMemberCrcAppBase
     public string? BankruptcyHistoryDetail { get; set; }
     public bool? IsTreatedForMHC { get; set; }
     public Address? ResidentialAddress { get; set; }
-    public Guid BizContactId { get; set; }
-    public Guid InviteId { get; set; }
 }
 
 
 #region authenticated
 public record ControllingMemberCrcUpsertCommand(ControllingMemberCrcAppUpsertRequest ControllingMemberCrcAppUpsertRequest) : IRequest<ControllingMemberCrcAppCommandResponse>;
 public record ControllingMemberCrcSubmitCommand(ControllingMemberCrcAppUpsertRequest ControllingMemberCrcUpsertRequest)
     : ControllingMemberCrcUpsertCommand(ControllingMemberCrcUpsertRequest), IRequest<ControllingMemberCrcAppCommandResponse>;
+public record ControllingMemberCrcAppUpdateCommand(
+    ControllingMemberCrcAppUpdateRequest ControllingMemberCrcAppRequest,
+    IEnumerable<LicAppFileInfo> LicAppFileInfos)
+    : IRequest<ControllingMemberCrcAppCommandResponse>;
 public record ControllingMemberCrcAppUpsertRequest : ControllingMemberCrcAppBase
 {
     public IEnumerable<Document>? DocumentInfos { get; set; }
     public Guid? ControllingMemberAppId { get; set; }
     public Guid? ApplicantId { get; set; }
+    public Guid BizContactId { get; set; }
+    public Guid InviteId { get; set; }
+    public Guid? ParentBizLicApplicationId { get; set; }
 };
 #endregion
 #region anonymous user
 public record ControllingMemberCrcAppSubmitRequest : ControllingMemberCrcAppBase
 {
+    public Guid? ControllingMemberAppId { get; set; }
+    public Guid? ApplicantId { get; set; }
```

Comment:
I think we talked about this. the applicantId should not be here.

Reply:
fe doesn't need to fill that, I will remove it from request object later